zebra-network/src/constants.rs

```diff
@@ -21,6 +21,29 @@ use zebra_chain::{
     serialization::Duration32,
 };
 
+/// The name of the current Zebra release.
+//
+// TODO: generate this from crate metadata (#2375)
+pub const RELEASE_NAME: &str = "Zebra:1.0.0-rc.6";
+
+/// The date of the current Zebra release.
+///
+/// This must be a valid `chrono::DateTime` with timezone string.
+pub const RELEASE_DATE: &str = "2023-03-23 00:00:00 +00:00";
+
+/// The maximum number of days after `RELEASE_DATE` where a Zebra server runs.
+///
+/// Notes:
+///
+/// - Zebra will exit with a panic if the current date is bigger than the `RELEASE_DATE` date plus this number of days.
+pub const EOS_PANIC_AFTER: u64 = 180;
+
+/// The number of days before the end of support where Zebra will display warnings.
+pub const EOS_WARN_AFTER: u64 = EOS_PANIC_AFTER - 5;
```

Comment:
`zcashd` uses 14 days here, some people need more than a week to plan downtime and an upgrade:
https://github.com/zcash/zcash/blob/38f7ed3ce4b9f7b678ab8ed93bbd911d8889d8e0/src/deprecation.h#L15

```suggestion
pub const EOS_WARN_AFTER: u64 = EOS_PANIC_AFTER - 14;
```